.gitignore

```diff
@@ -1,3 +1,9 @@
 coverage/
 node_modules/
 test/output/
+
+# ignore nyc output
+.nyc_output
+
+# ignore history
+.history
```

Comment:
These changes in .gitignore are OK, but can you create a separate PR for them? I prefer to have self-contained PRs.

Reply:
i see sorry

Reply:
No problem, thank you for working on the PR!

BTW, you don't need to close and create a new one, you can also just update an existing PR.